.github/CODEOWNERS

```diff
@@ -1 +1 @@
-@Hygieia/hygieia-iac-terraform-collector
+@Hygieia/hygieia-helm-terraform-collector
```

Comment:
the github admin team for this collector is hygieia-helm-collector, please change it to `@Hygieia/hygieia-helm-collector`